src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -1523,6 +1539,8 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                     });
                 }
             }
+            
+            LogTaskToAudit($"delete-{typeStr}", id, configuration: null);
```

Comment:
we want to see the task name here as well

Reply:
Done